.github/dependabot.yml

```
@@ -0,0 +1,19 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions" # See documentation for possible values
+    directory: "/" # Location of package manifests
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "pip" # See documentation for possible values
+    directory: "python/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "docker"
+    directory: "docker/"
```

```suggestion
    directory: "/"
```

Same as the `pip` one, seems to me that the `Dockerfile` is in the root directory.